helm/yatai-image-builder/values.yaml

```diff
@@ -112,8 +90,6 @@ bentoImageBuildEngine: kaniko # options: kaniko, buildkit, buildkit-rootless
 
 addNamespacePrefixToImageName: false
 
-juicefsStorageClassName: juicefs-sc
```

Comment:
we can not delete it

Reply:
I just removed the value and hard coded this, do you think we shouldn't?

Reply:
this need to be configurable, but we can set it to empty here and in the template, if the value is empty, set `juicefs-sc` to default

Reply:
That's fine. Why does it need to be configurable? Can we just drop support for legacy bentos?

Reply:
Because if juicefs data need to be re-build, we need to create a new juicefs storage and delete the old one, then change the name here. 
And we can not drop it because legacy bentos relay on juicefs and running.

Reply:
Ok, I'll just move it to internal then.